core/src/test/scala/org/apache/spark/sql/delta/UpdateScalaSuite.scala

```diff
@@ -60,6 +59,23 @@ class UpdateScalaSuite extends UpdateSuiteBase  with DeltaSQLCommandTest {
       Row(100, 101) :: Row(100, 101) :: Row(3, 30) :: Row(4, 40) :: Nil)
   }
 
+  test("update usage test - with space in path") {
+    val tmpDirForTest = System.getProperty("java.io.tmpdir").concat("/test with space/")
+    val dirFile = Utils.createTempDir(tmpDirForTest, "spark")
+    val dir = dirFile.toString
+    val df = Seq((1, 10), (2, 20), (3, 30), (4, 40)).toDF("key", "value")
+    val writer = df.write.format("delta").mode("append")
+    writer.save(dir)
+    spark.sql("CREATE TABLE test USING DELTA LOCATION '" + dir + "'")
```

Comment:
The table `test` needs to be dropped when the test finishes. Actually, you don't need to create a table, you can use `delta.<path>` to write the update command, such as

```
s"UPDATE delta.`${dir.getCanonicalPath}` SET key = key + 10"
```